src/main/java/io/netty/incubator/codec/quic/QuicheQuicChannel.java

```diff
@@ -333,6 +339,7 @@ void forceClose() {
             }
             connection = null;
             state = CLOSED;
+            timedOut = Quiche.quiche_conn_is_timed_out(conn.address());
```

Comment:
Is this method guaranteed to be called for all closures? If not, should we query and cache this `boolean` on demand if the connection is closed?

```java
        final QuicheQuicConnection conn = this.connection;
        // modify `timedOut` to a `byte`
        if (conn.isClosed() && timedOut == -1) {
            timedOut = Quiche.quiche_conn_is_timed_out(conn.address()) ? (byte) 0b1 : (byte) 0;
        }
        return timedOut == 1;
```

Reply:
yes its guaranteed.